errata_tool/product.py

```diff
@@ -55,6 +57,20 @@ def render(self):
             ],
         }
 
+    def releases(self):
+        """Get the list of product releases for this Product
+
+        :returns: a (possibly-empty) list of Release objects.
+        """
+        url = '/api/v1/releases?filter[product_id]=%s' % self.id
+        result = self._get(url)['data']
+        releases = []
+
+        for release in result:
+            release_name = release['attributes']['name']
+            releases.append(errata_tool.Release(id=release_name, data=release))
```

Comment:
Should this use the `name` kwarg instead of `id` to construct the `Release` object?

Reply:
@ktdreyer Oh, good catch!